gazebo_plugins/src/gazebo_ros_planar_move.cpp

```diff
@@ -139,7 +139,81 @@ namespace gazebo
     x_ = 0;
     y_ = 0;
     rot_ = 0;
+    linear_cmd_ = ignition::math::Vector3d();
+    angular_cmd_ = ignition::math::Vector3d();
+    last_linear_cmd_ = ignition::math::Vector3d();
+    last_angular_cmd_ = ignition::math::Vector3d();
     alive_ = true;
+    recover_roll_velocity_p_gain_  = 100; /*10.0;*/
+    recover_pitch_velocity_p_gain_ = 100; /*10.0;*/
+    recover_z_velocity_p_gain_ = 1000; /*1.0;*/
+
+    if (sdf->HasElement("recover_roll_velocity_p_gain"))
+      (sdf->GetElement("recover_roll_velocity_p_gain")->GetValue()->Get(recover_roll_velocity_p_gain_));
+    if (sdf->HasElement("recover_pitch_velocity_p_gain"))
+      (sdf->GetElement("recover_pitch_velocity_p_gain")->GetValue()->Get(recover_pitch_velocity_p_gain_));
+    if (sdf->HasElement("recover_z_velocity_p_gain"))
+      (sdf->GetElement("recover_z_velocity_p_gain")->GetValue()->Get(recover_z_velocity_p_gain_));
+    ROS_INFO("PlanarMovePlugin (ns = %s) recover_roll_velocity_p_gain_ = %f", robot_namespace_.c_str(), recover_roll_velocity_p_gain_);
+    ROS_INFO("PlanarMovePlugin (ns = %s) recover_pitch_velocity_p_gain_ = %f", robot_namespace_.c_str(), recover_pitch_velocity_p_gain_);
+    ROS_INFO("PlanarMovePlugin (ns = %s) recover_z_velocity_p_gain_ = %f", robot_namespace_.c_str(), recover_z_velocity_p_gain_);
+
+    x_velocity_limit_max_ = 1.0;
+    x_velocity_limit_min_ =-1.0;
+    y_velocity_limit_max_ = 1.0;
+    y_velocity_limit_min_ =-1.0;
+    rot_velocity_limit_max_ = 1.0;
+    rot_velocity_limit_min_ =-1.0;
+    x_acceleration_limit_max_ = 1.0;
+    x_acceleration_limit_min_ =-1.0;
+    y_acceleration_limit_max_ = 1.0;
+    y_acceleration_limit_min_ =-1.0;
+    rot_acceleration_limit_max_ = 1.0;
+    rot_acceleration_limit_min_ =-1.0;
+
+    if (sdf->HasElement("x_velocity_limit_max"))
+      (sdf->GetElement("x_velocity_limit_max")->GetValue()->Get(x_velocity_limit_max_));
```

Comment:
Why do you use these brackets around a statement?

Reply:
Wasn't me, that was from the original fix. If they do nothing we can probably remove them.